step-generation/src/__tests__/replaceTip.test.ts

```diff
@@ -132,6 +138,44 @@ describe('replaceTip', () => {
         }),
       ])
     })
+    it('Single-channel: dropping tips in waste chute', () => {
+      invariantContext = {
+        ...invariantContext,
+        additionalEquipmentEntities: {
+          wasteChuteId: {
+            name: 'wasteChute',
+            id: wasteChuteId,
+            location: 'D3',
```

Comment:
since we migrated over to v4 deck migration, lets update the location to reflect a true `cutoutId`:

```suggestion
            location: 'cutoutD3',
```